packages/edit-site/src/components/add-template/index.js

```diff
@@ -1,91 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { useSelect, useDispatch } from '@wordpress/data';
```

Comment:
Interesting, so instead of creating an arbitrary template, we would now have to override the base template level-by-level on a certain page type until we get to the desired specificity? I imagine we'll want to make it easier to "zoom in really far" in one go in the future

Reply:
There are only two levels of specificity:

- Page type, e.g., category, page, post.
- Page-specific, e.g., a specific category, specific page, specific post.